src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,29 @@
+import cn from 'classnames';
+import { ErrorMessage } from '../../types/ErrorMessage';
+
+type Props = {
+  errorNotification: ErrorMessage;
+  isHidden: boolean;
+  setIsHidden: (status: boolean) => void;
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  errorNotification,
+  isHidden,
+  setIsHidden,
+}) => {
+  return (
```

Comment:
```suggestion
}) => (
```

Reply:
so as I added handler on hidden class, I'd left it as it is